README.md

```diff
@@ -11,7 +11,7 @@ from tripleo-quickstart here.
 
 - CentOS 7
 - ideally on a bare metal host
-- user with passwordless sudo access
+- run as user 'stack' with passwordless sudo access
```

Comment:
This is fine, but you can run it as some other user if we add the provision/remote role to metalkube-setup-playbook.yml, or we set non_root_user to $USER before running ansible.

Whatever works at this point though :)